sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs

```diff
@@ -569,12 +687,12 @@ public partial class ProjectEnvironmentTypeCollection : Azure.ResourceManager.Ar
     public partial class ProjectEnvironmentTypeData : Azure.ResourceManager.Models.TrackedResourceData
```

Comment:
Is `DevCenterProjectEnvironment` better?

Reply:
Updated